assets/univ.js

```diff
@@ -80,9 +83,11 @@ function process_education_input() {
 
     var total = Object.keys(document.univ_courses).length;
     var msg = 'You finished ' + found + ' courses out of ' + total + '.';
-    if (not_found) {
+    if (not_found.length()) {
         msg += "<br>You also finished the following courses that I know nothing about: " + not_found.join(', ');
-
+    }
+    if (course_credits) {
+        msg += "<br>Your coursework cost at least " + course_credits + " credits.";
```

Comment:
I find the wording a bit weird (possibly because `cost` can be both present tense and past tense). How about *You have spent at least $C credits for your courses*?